docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -83,7 +83,7 @@ Java agent plugins are all pluggable. Optional plugins could be provided in `opt
 For using these plugins, you need to put the target plugin jar file into `/plugins`.
 
 Now, we have the following known optional plugins.
-* [Trace Spring beans](agent-optional-plugins/Spring-bean-plugins.md)
+* [Trace Spring beans](agent-optional-plugins/Spring-annotation-plugin.md)
```

Comment:
You should change the text of this link. Maybe `Spring annotation plugin` too?

Reply:
Done, Changed to Trace Spring annotation beans